requirements.txt

```diff
@@ -1,3 +1,3 @@
 ansicolors==1.0.2
-PyYAML~=4.2b4
-wheel==0.24.0
+PyYAML<6
```

Comment:
>5 as well?

Reply:
Since it used to be 4.2, I think almost anything would work.